server/views/proposedAddress/curfewAddressConfirmationForm.pug

```diff
@@ -0,0 +1,50 @@
+extends ../layout
+
+block content
+
+    div.pure-g.pure-u-1.borderBottom
+        div.back-link-container.smallPaddingTop
+            a.link-back(href="javascript: window.history.back();") Back
+
+        h2.heading-large Proposed curfew address
+
+        p The following discharge addresshave been requested by the offender.
+
+        include ./includes/preferredAddress
+
+        h3.heading-medium Address check confirmation
+
+
+        form(method="post")
+            input(type="hidden" name="_csrf" value=csrfToken)
+            input(type="hidden" name="nomisId" value=nomisId || '')
+            div.pure-g.largePaddingBottom
+                div.pure-u-1.pure-u-sm-5-12.pure-g
+                    div.form-group.inline
+                        p Does the homeowner/landlord give informed consent to HDC at the address?
+
+                        div.multiple-choice(data-target="landLordHtcConsentForm")
+                            input#hdcConsentYes(type="radio" checked=data && data.landLordConsent === 'Yes' name="landLordHDCConsent" value="Yes")
```

Comment:
There should always be data as default is {}

Reply:
throughout this page

Reply:
fair point

Reply:
- curfewAddressForm
- confirmAddressForm
- confirmationForm
- curfewAddressConfirmationForm
- submitForm

I'm lost...